easybuild/easyconfigs/n/numexpr/numexpr-2.7.0-intel-2019a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'numexpr'
+version = '2.7.0'
+
+homepage = 'https://numexpr.readthedocs.io'
+description = """The numexpr package evaluates multiple-operator array expressions many times faster than NumPy can.
+ It accepts the expression as a string, analyzes it, rewrites it more efficiently, and compiles it on the fly into
+ code for its internal virtual machine (VM). Due to its integrated just-in-time (JIT) compiler, it does not require a
+ compiler at runtime."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
```

Comment:
Is numexpr getting compiled with Intel VML? It makes a *huge* performance difference. We absolutely want this.

Reply:
@Micket How do I check/tell?

Reply:
`python -c "import numexpr; print(numexpr.use_vml)"`

Reply:
So, no:

```
$ python -c "import numexpr; print(numexpr.use_vml)"
False
```

We can include this as a sanity check command, but how to we force that `numexpr` is built with VML support? Any pointers on that?

Reply:
I did it manually once a while back when testing performance; I think I just put in the VML library in the site.cfg file. https://github.com/pydata/numexpr/blob/6a9f2ad1128a4f6577d2cb9967dab0f89698131c/site.cfg.example#L41

Reply:
OK, this will definitely require an easyblock for `numexpr`, since the `vml` library to specify depends on the processor architecture: `mkl_vml_avx`, `mkl_vml_avx2`, `libmkl_vml_avx`, etc. Goodie.